### PR TITLE
Update command outputs to k8s 1.13 based and split command outputs

### PR DESCRIPTION
**Pull request by makocchi-git**
- Add downloading a sample yaml
- Split command outputs
- Update kubectl outputs to k8s 1.13 based

### CODE REVIEW

**Comment by makocchi-git** [comment]
[preview is here](https://deploy-preview-13014--kubernetes-io-master-staging.netlify.com/docs/tasks/job/parallel-processing-expansion/)
[original](https://kubernetes.io/docs/tasks/job/parallel-processing-expansion/)

File: content/en/docs/tasks/job/parallel-processing-expansion.md [hunk 1]
```diff
@@ -54,6 +57,9 @@ Check if it worked:
 
 ```shell
 ls jobs/
+```
+
+```
```
Comment (DanyC97) [inline content/en/docs/tasks/job/parallel-processing-expansion.md]: please  add `The output is similar to this:` as per the guidelines before any output

File: content/en/docs/tasks/job/parallel-processing-expansion.md [hunk 2]
```diff
@@ -67,19 +73,25 @@ Next, create all the jobs with one kubectl command:
 
 ```shell
 kubectl create -f ./jobs
-job "process-item-apple" created
-job "process-item-banana" created
-job "process-item-cherry" created
+```
+
```
Comment (DanyC97) [inline content/en/docs/tasks/job/parallel-processing-expansion.md]: please add The output is similar to this: as per the guidelines before any output

File: content/en/docs/tasks/job/parallel-processing-expansion.md [hunk 3]
```diff
@@ -67,19 +73,25 @@ Next, create all the jobs with one kubectl command:
...
 Now, check on the jobs:
 
 ```shell
 kubectl get jobs -l jobgroup=jobexample
-NAME                  DESIRED   SUCCESSFUL   AGE
-process-item-apple    1         1            31s
-process-item-banana   1         1            31s
-process-item-cherry   1         1            31s
+```
+
```
Comment (DanyC97) [inline content/en/docs/tasks/job/parallel-processing-expansion.md]: please add The output is similar to this: as per the guidelines before any output

File: content/en/docs/tasks/job/parallel-processing-expansion.md [hunk 4]
```diff
@@ -90,6 +102,9 @@ We can check on the pods as well using the same label selector:
 
 ```shell
 kubectl get pods -l jobgroup=jobexample
+```
+
```
Comment (DanyC97) [inline content/en/docs/tasks/job/parallel-processing-expansion.md]: please add The output is similar to this: as per the guidelines before any output

File: content/en/docs/tasks/job/parallel-processing-expansion.md [hunk 5]
```diff
@@ -104,6 +119,9 @@ for p in $(kubectl get pods -l jobgroup=jobexample -o name)
 do
   kubectl logs $p
 done
+```
+
```
Comment (DanyC97) [inline content/en/docs/tasks/job/parallel-processing-expansion.md]: please add The output is similar to this: as per the guidelines before any output

**Comment by makocchi-git** [comment]
@DanyC97 Thank you for your comments! I fixed.

File: content/en/docs/tasks/job/parallel-processing-expansion.md [hunk 6]
```diff
@@ -104,6 +127,11 @@ for p in $(kubectl get pods -l jobgroup=jobexample -o name)
...
+The output is:
```
Comment (DanyC97) [inline content/en/docs/tasks/job/parallel-processing-expansion.md]: nit: for consistency reason, would you agree is good to keep `similar to this:` in ?

File: content/en/docs/tasks/job/parallel-processing-expansion.md [hunk 7]
```diff
@@ -67,19 +75,29 @@ Next, create all the jobs with one kubectl command:
 
 ```shell
 kubectl create -f ./jobs
-job "process-item-apple" created
-job "process-item-banana" created
-job "process-item-cherry" created
+```
+
+The output is:
```
Comment (DanyC97) [inline content/en/docs/tasks/job/parallel-processing-expansion.md]: nit: for consistency reason, would you agree is good to keep `similar to this:` in ?

**Comment by makocchi-git** [comment]
@DanyC97 Thanks again! I fixed.

**Comment by DanyC97** [comment]
@cody-clark please help review

**Comment by makocchi-git** [comment]
@cody-clark ping

**Comment by DanyC97** [comment]
@jaredbhatti please help review

**Comment by makocchi-git** [comment]
@jaredbhatti ping

**Comment by makocchi-git** [comment]
@cody-clark @jaredbhatti ping

**Pull request merged**